homeassistant/components/twinkly/light.py

```diff
@@ -185,16 +185,47 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                 if isinstance(self._attr_rgbw_color, tuple):
 
                     await self._client.interview()
-                    # Static color only supports rgb
-                    await self._client.set_static_colour(
-                        (
-                            self._attr_rgbw_color[0],
-                            self._attr_rgbw_color[1],
-                            self._attr_rgbw_color[2],
+                    if LightEntityFeature.EFFECT in self.supported_features:
```

Comment:
`in` is not the correct check. `supported_features` is an integer. Calling `in` on an integer raises a `TypeError`.

Reply:
Ah, I see now that this is supported because supported features inherits from `IntFlag`.

Reply:
Sorry, thought it was a LightEntityFeature, which again is a enum.IntFlag.  
Changed to & now throughout the code for consitency.

Reply:
Missed your second comment.  Should I add it back as "in" or do you prefer &?
Personally I think "in" is easier to read, but that is just my opinion.

Reply:
`in` is easier to read once you understand it's a flag 😅 I don't really care.